src/main/target/STM32_UNIFIED/target.h

```diff
@@ -333,4 +333,11 @@
 #define USE_RANGEFINDER
 #define USE_RANGEFINDER_HCSR04
 #define USE_RANGEFINDER_TF
+
+#define USE_RX_EXPRESSLRS
+#define USE_RX_SX1280
+#if !defined(STM32F745)
+#define USE_RX_SX127X
```

Comment:
What's the reason for not building SX127x support for F745 target?

Reply:
It did not fit into memory a few months ago, I'll try to remove it and maybe it'll fit.

Reply:
Removed `ifdef` and it is building with `SX127X` drivers compiled in.